### ISSUE TITLE
could avoid intrioducing MLStyle.jl

### ISSUE THREAD

**Issue by thautwarm** [shipengcheng1230/GmshTools.jl#6]
You're using an older version of MLStyle, and now many other packages are depending on MLStyle v0.4+, whch will lead to the failure of resolving your package versions.
 
Your use of MLStyle seems simple enough, so I think there is no need to introduce an extra dependency.
 
https://github.com/shipengcheng1230/GmshTools.jl/blob/ff95d2851506c75fc311a7fd019e69a636c88fc1/src/macros.jl#L26

There is no dispatch on data but only on types, hence using idiomatic Julia code can be regarded as the better.
```julia
match_tuple(e::Expr) = e.head == :tuple ? e : nothing
match_tuple(a) = nothing
```

**Comment by shipengcheng1230** [comment]
Thanks for bringing it up!

**Comment by shipengcheng1230** [comment]
fixed by #7
